students/kylee/users/validator.py

```diff
@@ -0,0 +1,24 @@
+import re
+
+from django.core.exceptions   import ValidationError
+from django.utils.translation import gettext_lazy as _
+
+def email_validate(email) :
+    if not re.match('^[\w+-\_.]+@[\w]+\.[\w]+$', email) :
+        raise ValidationError(
+            _(' 입력하신 이메일 %(email)s 를 확인해주세요. 이메일은 @와 . 이 순서대로 들어가야 합니다.'),
+            params={'email':email}
+        )
+
+def password_validate(password) :
```

Comment:
함수명은 반드시 동사로 시작하도록 만들어주세요. 함수는 action 입니다. 


```suggestion
def validate_email(email) :
    if not re.match('^[\w+-\_.]+@[\w]+\.[\w]+$', email) :
        raise ValidationError(
            _(' 입력하신 이메일 %(email)s 를 확인해주세요. 이메일은 @와 . 이 순서대로 들어가야 합니다.'),
            params={'email':email}
        )

def validate_password(password) :
```